infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LogReplicationServer.java

```diff
@@ -56,10 +57,19 @@ public class LogReplicationServer extends AbstractServer {
 
     private static final String EXECUTOR_NAME_PREFIX = "LogReplicationServer-";
 
-    private Map<ReplicationSession, LogReplicationSinkManager> sourceSessionToSinkManagerMap = new HashMap<>();
+    private Map<LogReplication.ReplicationSessionMsg, LogReplicationSinkManager> incomingSessionMsgToSinkManagerMap = new HashMap<>();
```

Comment:
I tried to not use ReplciationSession because I think in the coming SessionManger PR, the ReplicationSession class is removed and we have only the proto msg.

Keeping that in mind, mindfully used only the proto msg while extracting this bit out from the connection-model PR.